package.json

```diff
@@ -37,9 +39,18 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.21.2",
     "eslint-plugin-prettier": "^3.1.4",
-    "nodemon": "^1.18.10",
+    "nodemon": "^2.0.5",
     "prettier": "^2.0.5",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11"
+  },
+  "gitHooks": {
```

Comment:
I can't seem to get this to work? Could you explain how to do it? @opensourceally 
I am able to use the npm library [pre-commit](https://www.npmjs.com/package/pre-commit)

Reply:
Maybe husky can also be used

Reply:
I will test with husky . I had taken the previous one from another repository without checking the pre-commit scripts. They were there in .git (hidden)folder. Will correct and test it this time around before PR.

Reply:
I see that the following errors are there  on linting.

D:\Hacktoberfest\CryptoFS\src\public\js\keymanagement.js
  1:19  warning  Unable to resolve path to module 'forge'  import/no-unresolved

D:\Hacktoberfest\CryptoFS\src\public\js\list.js
  35:19  error  'forge' is not defined  no-undef
  36:18  error  'forge' is not defined  no-undef
  37:22  error  'forge' is not defined  no-undef
  39:21  error  'forge' is not defined  no-undef

D:\Hacktoberfest\CryptoFS\src\public\js\mainpage.js
   1:19  warning  Unable to resolve path to module 'forge'  import/no-unresolved
  64:28  error    '$' is not defined                        no-undef
  66:5   error    'grecaptcha' is not defined               no-undef

D:\Hacktoberfest\CryptoFS\src\public\js\util.js
  35:5  error  Assignment to property of function parameter 'el'  no-param-reassign
  45:5  error  Assignment to property of function parameter 'el'  no-param-reassign

D:\Hacktoberfest\CryptoFS\src\routes\index.js
  27:29  error  Arrow function should not return assignment  no-return-assign

✖ 11 problems (9 errors, 2 warnings)

I also have tried with  "settings": {
    "import/resolver": {
      "node": {
        "extensions": [".js", ".jsx", ".ts", ".tsx"]
      }
    }
  }. But these seems to not work with import of forge.

Reply:
As of now husky is intoduced and it is giving me linting errors. Can I suppress the linter for now for the above changes and checkin ??

For this I need to modify  "import/no-unresolved": "off",
    "no-return-assign" : "off",
    "no-param-reassign" : "off",
    "no-undef":"off" on eslint.json rules


Please let me know if this is Ok?

Reply:
> Yes, for now, anything that can't be autofixed, can be left Regards, Priyansh Jain
> […](#)
> On Thu, 15 Oct 2020, 22:43 Prashanth Prabhu, ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In package.json <[#17 (comment)](https://github.com/Presto412/CryptoFS/pull/17#discussion_r505707633)>: > "prettier": "^2.0.5", "webpack": "^4.43.0", "webpack-cli": "^3.3.11" + }, + "gitHooks": { As of now husky is intoduced and it is giving me linting errors. Can I suppress the linter for now for the above changes and checkin ?? For this I need to modify "import/no-unresolved": "off", "no-return-assign" : "off", "no-param-reassign" : "off", "no-undef":"off" on eslint.json rules Please let me know if this is Ok? — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#17 (comment)](https://github.com/Presto412/CryptoFS/pull/17#discussion_r505707633)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AGJHFR5UPQIWYRVH63HWRZLSK4UUVANCNFSM4SQZFSBQ> .

Please check now. I have made the changes.